defold-rive/pluginsrc/com/defold/bob/pipeline/RiveSceneBuilder.java

```diff
@@ -14,47 +14,22 @@
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
+
+import com.dynamo.bob.*;
```

Comment:
🙀

Reply:
I should find an option in IDEA that turns it off =/